### ISSUE TITLE
[Question]Quantized QuartzNet

### ISSUE THREAD

**Issue by ABoltachev** [NVIDIA/NeMo#1425]
I am trying to quantize a QuartzNet 15x5 encoder like this:
```python
jasper_encoder = jasper_encoder.cpu()
jasper_encoder.eval()
jasper_encoder.qconfig = torch.quantization.get_default_qconfig('qnnpack')
torch.quantization.propagate_qconfig_(jasper_encoder)
jasper_encoder = torch.quantization.prepare(jasper_encoder, inplace=False)
jasper_encoder = torch.quantization.convert(jasper_encoder)
```
But, unfortunately, the encoder is not fully quantized, but only some RuLu. I also tried to quantize MaskedConv1d separately, but unfortunately even in this case the Conv1d layer was not quantized
This is what I get when I do print (jasper_encoder.encoder) after quantizing:
```
Sequential(
  (0): JasperBlock(
    (mconv): ModuleList(
      (0): MaskedConv1d(
        (conv): Conv1d(64, 64, kernel_size=[33], stride=[2], padding=(16,), dilation=[1], groups=64, bias=False)
      )
      (1): MaskedConv1d(
        (conv): Conv1d(64, 256, kernel_size=(1,), stride=(1,), bias=False)
      )
      (2): BatchNorm1d(256, eps=0.001, momentum=0.1, affine=True, track_running_stats=True)
    )
    (mout): Sequential(
      (0): QuantizedReLU()
      (1): Dropout(p=0.0, inplace=False)
    )
  )
...
```
As you can see from this output, neither Conv1d nor BatchNorm1d were quantized. Could you help me with quantizing QuartzNet if it is possible to quantize this model

**Environment overview (please complete the following information)**

 - Environment location: Bare-metal
 - Method of NeMo install: pip install

**Environment details**
 - OS version: Ubuntu 18.04 LTS
 - PyTorch version : 1.6.0
 - Python version: Python 3.6
 - NeMo version: 0.10.1

**Comment by khursani8** [comment]
I don't think they can help with this
Pytorch need to support Conv1d and BatchNorm1d in order for you to quantize it.

The only way you can do it is by converting Conv1d to Conv2d(same for BatchNorm) using [this](https://github.com/NVIDIA/NeMo/blob/c90e3aaee67a73dc1c2b0c0614766246e403e1c1/nemo/utils/export_utils.py#L53)

That code not working right now, you need to change some stuff to make it work

**Comment by okuchaiev** [comment]
we are actually working on adding quantization support, but it is not there yet.

**Comment by ABoltachev** [comment]
I apologize, I was wrong with the PyTorch version, I used 1.4.0 but have now updated to 1.6.0
The problem with Conv1d is gone as it is supported in version 1.6.0. Also the issue with BatchNorm1d has been resolved (https://github.com/pytorch/pytorch/issues/41642). However, now, I have a problem with ReLU.
For some reason, not all ReLUs are quantized and, most likely because of this, at the trace stage in the line https://github.com/NVIDIA/NeMo/blob/v0.10.1/nemo/collections/asr/parts/jasper.py#L425 throws "RuntimeError: _Map_base :: at" exception
